src/universal/utils/isObject.js

```diff
@@ -1,3 +1,3 @@
 export default function isObject(val) {
-  return val && typeof val === 'object';
+  return val && typeof val === 'object' && !(val instanceof Date);
```

Comment:
This is a sneaky little change. Perhaps method should be isObjectOtherThanDate() 😉 

If we add more types, perhaps this method could become `isObject(val, except = [class0, class1, ...])`